### PR TITLE
[RDY] Add title/subtitle to openpane

### PR DESCRIPTION
**Pull request by ohuang12**
Adds some text to the open file pane to make it more user friendly.

### CODE REVIEW

File: dnainator-javafx/src/main/java/nl/tudelft/dnainator/javafx/controllers/FileOpenController.java [hunk 1]
```diff
@@ -39,6 +40,7 @@
 	private static final int WIDTH = 550;
 	private static final int ANIM_DURATION = 250;
 
+	@SuppressWarnings("unused") @FXML private AnchorPane container;
 	@SuppressWarnings("unused") @FXML private GridPane fileOpenPane;
```
Comment (Unia) [inline dnainator-javafx/src/main/java/nl/tudelft/dnainator/javafx/controllers/FileOpenController.java]: All `fileOpenPane` uses should be replaced with `container`. When that's done, `fileOpenPane` can be removed.

**Comment by Unia** [comment]
Nice :) Can you also, while you're digging in the fxml now anyway, fix the vertical offset of the buttonbar in the welcome screen? It's been bugging me more than it should have...

File: dnainator-javafx/src/main/java/nl/tudelft/dnainator/javafx/controllers/FileOpenController.java (hunk 1 above)
Comment (ohuang12) [inline dnainator-javafx/src/main/java/nl/tudelft/dnainator/javafx/controllers/FileOpenController.java]: Yep will do that!

**Comment by ohuang12** [comment]
Sure I'll have a look at it.

**Comment by Unia** [comment]
Nice! Next reviewer may merge!

**Comment by pvanagtmaal** [comment]
LGTM. Waiting for travis, will merge after.

**Pull request merged**